sdk/storage/src/table/clients/entity_client.rs

```diff
@@ -353,3 +363,85 @@ mod integration_tests {
         // TODO: Confirm that the entity was merged
     }
 }
+
+pub struct PipelineEntityClient {
```

Comment:
Is there a reason we have an entirely new struct? In Cosmos we are slowly moving all the existing clients over to use pipelines instead of creating new clients.

Reply:
The only reason was to give me a way to check the changes without deleting an existing code. From now on I will begin to delete duplicate code even if it is not completely finished